src/components/Card/Card.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../styles/common';
 
 .Card {
-  overflow: hidden;
+  overflow: var(--p-override-visible, hidden);
```

Comment:
This is causing an issue with the focus ring. I'm thinking since we are since in development we change this to overflow visible to see if this breaks anything. @danrosenthal @kyledurand what do you think?

Reply:
I'm a fan over removing this entirely. I think @dfmcphee had to port the card component in web just to remove this property. Might be best to do this in a separate PR though, for change log visibility

Reply:
This'll probably be considered a breaking-ish change. So we might want to wait until v5 for it to be the default. I think it makes sense to use the fallback approach until then.

This may have repercussions on other components like Resource List, which goes full width in a card. We might need to add a bunch of extra border radius to that like we've done in polaris rails.